index.bs

```diff
@@ -1615,7 +1615,7 @@ Unlike dictionaries, interfaces:
     (i.e., one can check if  it is an `instanceof` a particular class on the global scope),
 
 Defining an interface also exposes it on the global scope, allowing for the specification of static methods.
-For example, the `canParse()` static method of the URL interface. 
+For example, the `canParse()` static method of the URL interface.
```

Comment:
Unrelated change, but it's something many editors do automatically. I can remove it, but I imagine others will run into the same thing.

Reply:
You are not the only one.  We should maybe lint for trailing whitespace.